<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PriortisationDialogBase</class>
 <widget class="QDialog" name="PriortisationDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Indian WAtershed Management support System</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="PriortisationDialog">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Priortisation">
      <attribute name="title">
       <string>Priortisation</string>
      </attribute>
      <widget class="QProgressBar" name="PG_PR">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>330</y>
         <width>251</width>
         <height>21</height>
        </rect>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
      <widget class="QLineEdit" name="LE_PR_2">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>270</y>
         <width>201</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="LE_PR_1">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>200</y>
         <width>201</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_7">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>90</y>
         <width>281</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Area under unassured irrigation</string>
       </property>
      </widget>
      <widget class="QLabel" name="LB_PR_2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>270</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Output Folder</string>
       </property>
      </widget>
      <widget class="QLabel" name="LB_SR_1">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>210</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Input Folder</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_4">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>50</y>
         <width>271</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Proportion of ST/SC</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>30</y>
         <width>261</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Preponderance of wastelands</string>
       </property>
      </widget>
      <widget class="QPushButton" name="PB_PR_MR">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>320</y>
         <width>101</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Model Run</string>
       </property>
      </widget>
      <widget class="QPushButton" name="PB_PR_1">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>200</y>
         <width>75</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_9">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>90</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Soil erosion rates</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>30</y>
         <width>271</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Acutness of drinking water</string>
       </property>
      </widget>
      <widget class="QPushButton" name="PB_PR_2">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>260</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_3">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>50</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Exploitation of ground water</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_5">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>70</y>
         <width>281</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Productivity potential</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_10">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>110</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Select All</string>
       </property>
      </widget>
      <widget class="QLabel" name="LB_SR_2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>0</y>
         <width>471</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="text">
        <string>Select criteria for priortisation</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_8">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>110</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Willingness of community</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_6">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>70</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Contiguity to treated units</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
